### DISCUSSION TITLE
How to get the coordinates of a prediction?

### DISCUSSION THREAD

**Discussion by OBaruch** [hukaixuan19970627/yolov5_obb#490] [Q&A]
After executing an inference whit: 

python detect.py --weights 'runs/train/yolov5m_csl_dotav1.5/weights/best.pt' \
  --source 'dataset/dataset_demo/images/' \
  --img 2048 --device 0 --conf-thres 0.25 --iou-thres 0.2 --hide-labels --hide-conf

how to get the bound box coordinates of the predictions?

**Comment by quangsonle** [comment]
i have the same question, it is not like the conventional yolo where the bounding box is opencv's rectangle (defined by a center and width and length), i suppose the output's dimension here is longer. 

How is your status, any relevant findings?
